apm-agent-plugins/apm-jdbc-plugin/pom.xml

```diff
@@ -15,7 +15,7 @@
         <dependency>
             <groupId>p6spy</groupId>
             <artifactId>p6spy</artifactId>
-            <version>3.6.0</version>
+            <version>3.7.0-SNAPSHOT</version>
```

Comment:
Isn't this risky? You should use a real version. If a snapshot is a must, which is not recommended, at least resolve the snapshot and use the version with the timestamp so at least you have a reproducible build

Reply:
That’s why it says wip in the title ;)